src/Files.Uwp/Strings/en-US/Resources.resw

```diff
@@ -2795,4 +2795,10 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="ReviewFilesContent" xml:space="preserve">
     <value>Would you like to review Files?</value>
   </data>
+  <data name="SetAsLockscreen" xml:space="preserve">
+    <value>Set as lockscreen background</value>
+  </data>
+  <data name="SetAsText" xml:space="preserve">
+    <value>Set as desktop background</value>
```

Comment:
Are there existing strings for this?

Reply:
Yeah, there probably are. I'll take a look.

Reply:
What about "Set as wallpaper" with the sub options being "Desktop", "Lockscreen", and "Both"?

Reply:
That sounds like a good idea. I'll make a separate PR for the context menu.